src/pieces/commands/commands_functions.py

```diff
@@ -0,0 +1,474 @@
+## MAIN FUNCTIONS | Line ~33
+## HELPER FUNCTIONS | Line ~381
+
+from pieces.gui import *
```

Comment:
yeah I think we to do that organisations that I made that were 2 files `api.py`, `commands.py` they were too messy with almost 500 lines of code or more so I made instead of the commands.py and the api.py a package we can still break the commads_functions into main and helper functions files we can do that